Kudu.Core/Deployment/WellKnownEnvironmentVariables.cs

```diff
@@ -38,5 +38,9 @@ internal static class WellKnownEnvironmentVariables
 
         public const string GoWebConfigTemplate = "GO_WEB_CONFIG_TEMPLATE";
         public const string SelectLatestVersionCommandKey = "SELECT_LATEST_VERSION_CMD";
+
+        public const string GypMsvsVersion = "GYP_MSVS_VERSION";
+        public const string VCTargetsPath = "VCTargetsPath";
```

Comment:
nit: should the variables be prefixed as VC_TargetsPath, VC_InstallDir140, to match the pattern of other vars? (e.g. GO_)